addons/account/views/account_move_views.xml

```diff
@@ -478,7 +480,10 @@
                         <!-- Post -->
                         <button name="action_post" string="Post" class="oe_highlight"
                                 type="object" groups="account.group_account_invoice"
-                                attrs="{'invisible': [('state', '!=', 'draft')]}"/>
+                                attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('move_type', '!=', 'entry')]}"/>
```

Comment:
I know it's boring but we will probably add some move_type one day so I'll prefer
```
('move_type', 'in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))
```

Reply:
Not sure... We want to have either `Post`, either ``Validate`, so one needs to be the exact opposite of the other : `('move_type', '=', 'entry')` and `('move_type', '!=', 'entry')`. If we use your approach, we might end up with a type that you cannot validate/post

Reply:
Indeed... nevermind then